lib/src/themes/ayu_mirage.toml

```diff
@@ -12,13 +12,15 @@
 "comment" = { fg = "gray", modifiers = ["italic"] }
 "variable" = { fg = "foreground" }
 "variable.parameter" = { fg = "yellow" }
+"include" = { fg = "orange" }
```

Comment:
These are used by the haskell treesitter highlighting and are notably missing from the themes. It's maybe a hack that I put these in this one theme only though.